live-editing/configs/DateTimeEditorConfigGenerator.ts

```diff
@@ -0,0 +1,28 @@
+import { IgxDateTimeEditorDirective, IgxInputGroupModule } from "igniteui-angular";
+
+import { DateТimeЕditorSample1Component } from "../../src/app/scheduling/datetimeeditor/datetimeeditor-sample-1/datetimeeditor-sample-1.component";
+
```

Comment:
Add IConfigGenerator import

```suggestion
import { IConfigGenerator } from "./core/IConfigGenerator";
```